doc/changelog/08-vernac-commands-and-options/19872-depr-auto-prop-lowering.rst

```diff
@@ -0,0 +1,4 @@
+- **Removed:**
+  flag `Automatic Proposition Inductives` (using its effect was deprecated since 8.20)
```

Comment:
The option itself was not deprecated (`Set Automatic Proposition Inductives` gives no deprecation warning), nor do I see anything in the changelog or documentation saying that it was deprecated.  Shouldn't this instead be deprecated in 9.0 and then removed in 9.1 unless there's an urgent reason to remove it immediately?

Reply:
Its effect when on was deprecated. The option was there to provide the future behaviour (by unsetting the option). 
In other words relying on the option being on was deprecated. Now there is only the off behaviour.
So there is no point keeping it.